Patches/[O21] Outer Rim/Patch_OuterRim_Apparel.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	
+	<Operation Class="PatchOperationFindMod">
+    <mods>
+		<li>Outer Rim - Tatooine (Continued)</li>
+    </mods>
+		<match Class="PatchOperationSequence">
+		<operations>
+			
+			
+		<li Class="PatchOperationAdd">
+			<xpath>Defs/ThingDef[defName="O21_OuterRim_JawaRobe" or defName="O21_OuterRim_TuskenRaiderRobeA" or defName="O21_OuterRim_TuskenRaiderRobeB"]</xpath>
+			<value>
+				<equippedStatOffsets>
+				  <CarryBulk>60</CarryBulk>
```

Comment:
This can probably be reduced by half--the Jawas and Tuskens don't carry much equipment.

Reply:
I reduced it to 40 since this covers both backpack and webbing